TrafficReplayer/src/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -193,6 +233,43 @@ public Stream<byte[]> getResponseDataStream() {
         }
     }
 
+    public static class RequestResponseResponseJSONTriple {
```

Comment:
Move this to it's own class.  See the comments above about how this should be associated w/ the write callbacks, not the replayer.  That coupling makes the code less general and will make unit tests harder to author.

It's also worth noting that this is a stopgap solution that was put in place to expedite things for the traffic comparison engine.  Longer term, I would like to see the output of this to be as detailed as the data we have from the original dumps (notice that we could look to share the same netty WireHandler if we really wanted to).